configure.ac

```diff
@@ -25,6 +26,20 @@ AC_ARG_ENABLE([examples],
                               [examples="no"])
 AM_CONDITIONAL([EXAMPLES], [test x$examples = xyes])
 
+AC_ARG_ENABLE([tests],
+              [AS_HELP_STRING([--enable-tests],
+                              [Build Tests @<:@no@:>@])],
+                              [tests="$enableval"],
+                              [tests="no"])
+AM_CONDITIONAL([TESTS], [test x$tests = xyes ])
+AS_IF(
+    [test "xtests$" = "xyes"],
+    [PKG_CHECK_MODULES(
+      [catch2], [catch2 >= 1.6.37],
```

Comment:
we need to enforce some version on this also, any suggestions?

Reply:
Let's use the current version that we are using. I have set `catch2` allowed versions to be `>=2.13.5` for both automake and cmake.